src/app/components/store-products/store-products.component.ts

```diff
@@ -8,4 +16,23 @@ import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class StoreProductsComponent {
+  readonly search: FormGroup = new FormGroup({ searchProduct: new FormControl() });
+  readonly store$: Observable<StoreModel> = this._activatedRoute.params.pipe(switchMap(data => this._storesService.getOneStore(data["storeId"])),
+   shareReplay(1));
+
+   readonly startsWith$: Observable<string> = this.search.valueChanges.pipe(
```

Comment:
the name startsWith is a bit confusing. Maybe it should be called defaultSearchValue or something